sdks/python/run_postcommit.sh

```diff
@@ -88,18 +85,18 @@ python setup.py nosetests \
     --requirements_file=postcommit_requirements.txt \
     --num_workers=1"
 
-# Run wordcount on the Google Cloud Dataflow service
-# and validate job that finishes successfully.
-echo ">>> RUNNING TEST DATAFLOW RUNNER py-wordcount"
+# Run integration tests on the Google Cloud Dataflow service
+# and validate that jobs finish successfully.
+echo ">>> RUNNING TEST DATAFLOW RUNNER it tests"
 python setup.py nosetests \
   -a IT \
+  --processes=4 \
```

Comment:
We also need to add `--process-timeout` since the default value is ~10s if I remember correctly. Otherwise will raise TimedOutException before job finishes.